cmd/import.go

```diff
@@ -74,7 +79,66 @@ func evolveSchema(ctx context.Context, db string, coll string, docs []json.RawMe
 	return util.Error(err, "create or update collection")
 }
 
+func fixNumbers(ctx context.Context, coll string, docs []json.RawMessage) {
+	err := schema.Infer(&sch, coll, docs[0:1], PrimaryKey, AutoGenerate, 1)
+	util.Fatal(err, "infer schema")
+
+	if schema.HasArrayOfObjects {
```

Comment:
Is this issue only with `ArrayOfObjects` or does this affect `Objects` as well?

Reply:
Looks like with array of objects only. @himank can confirm?

Reply:
That's right.